app.config.ts

```diff
@@ -34,14 +33,13 @@ const appConfig: ExpoConfig & { extra: AppEnv & { NODE_ENV: AppStage; RELEASE_ID
   assetBundlePatterns: ['**/*'],
   runtimeVersion: packageJson.version,
   ios: {
+    jsEngine: 'jsc',
```

Comment:
we run on Android in Hermes and on IOS in JSC then, isn't it?

Reply:
correct, at least for now, Hermes works very well with low end devices on Android